index.md

```diff
@@ -1,73 +1,52 @@
-# Please Don't Just Say Hello In Chat
+# Please Don't Just Say Hello in Chat
 
 ## TL;DR
 
-Modern chat communication should be done in a way that saves time for all
-participants and enables an efficient, asynchronous exchange of information.
-**Don't expect an answer if you just write "Hello" in chat**. Instead, phrase a
-message that allows your chat partners to provide a meaningful response as soon
-as they allocate time to read and answer your message.
+Modern chat communication should be done in a way that saves time for all participants and enables an efficient, asynchronous exchange of information. Don’t expect an answer if you just write “Hello” in chat. Instead, phrase a message that allows your chat partners to provide a meaningful response as soon as they allocate time to read and answer your message.
 
 ## Chat Etiquette
 
-<pre>
+```text
 2010-07-19 12:32:12 you: Hi
 2010-07-19 12:32:15 co-worker: Hello.
 # CO-WORKER WAITS WHILE YOU PHRASE YOUR QUESTION
 2010-07-19 12:34:01 you: I'm working on [something] and I'm trying to do [etc...]
 2010-07-19 12:35:21 co-worker: Oh, that's [answer...]
-</pre>
+```
 
-It's as if you called someone on the phone and said "Hi!" and then put them on
-hold!
+It’s as if you called someone on the phone and said “Hi!” and then put them on hold!
```

Comment:
I'm not sure why to replace all `"` by `“`. The former is valid punctuation. I reckon it is more difficult to incorporate follow-up changes when sticking to this convention, so I'd rather reject it. Also applies to all the other changes in this PR.